.gitignore

```diff
@@ -28,6 +28,9 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules/
+!client/chrome-app/lib/ssh2-streams/node_modules/
```

Comment:
?
Why aren't we just installing these to node_modules and browserifying from there?
We included dependencies for our Java code because there wasn't a good way to do package management/dependencies.